panos_upgrade_assurance/check_firewall.py

```diff
@@ -949,10 +949,10 @@ def run_readiness_checks(
         for check in checks_list:
             if isinstance(check, dict):
                 check_type, check_config = next(iter(check.items()))
-                # check_result = self._check_method_mapping[check_type](check_config)
+                if check_config is None:
```

Comment:
I think we should include dicts with null value in the unittests.

Reply:
covered in [this commit](https://github.com/PaloAltoNetworks/pan-os-upgrade-assurance/pull/101/commits/706ff8d2ad34662f89b8f2e42f9d3bb90bb7c2bd)